PySDM/initialisation/sampling/spatial_sampling.py

```diff
@@ -8,23 +8,38 @@
 
 class Pseudorandom:  # pylint: disable=too-few-public-methods
     @staticmethod
-    def sample(*, backend, grid, n_sd, z_part=None):
-        dimension = len(grid)
-        n_elements = dimension * n_sd
+    def sample(*, backend, grid, n_sd, z_part=None, x_part=None):
+        n_dims = len(grid)
+        scale_fac = (
+            []
+        )  # scale factor for each dimension; y = scale_factor * eps + affine_factor
```

Comment:
@jtbuch, what is `eps`?

Reply:
`eps` is just my notation for an array of random uniform samples, which in this case is the `positions` array.

Reply:
Hi @slayoo : `eps` is just my notation for an array of random uniform samples, which in this case is the `positions` array.  Sorry, I should have been consistent with my notation.